src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-introduction.mdx

```diff
@@ -15,428 +15,108 @@ redirects:
 
-OpenTelemetry logs are compatible with logs in New Relic. The OpenTelemetry logs optionally include attributes (name-value pairs) and resource attributes that map directly to dimensions you can use to facet or filter log data with queries. OpenTelemetry log metadata (for example, `name`, `severity_text`, and `trace_id`) also map directly to dimensions on New Relic's <InlinePopover type="logs" /> capabilities. We currently support all OpenTelemetry log message types.
+We recommend that you explore both New Relic and OpenTelemetry options. With New Relic instrumentation, there are inherent advantages to developing instrumentation and platform features which work together, and New Relic integrations tend to work better out of the box. On the other side, OpenTelemetry offers an unparalleled degree of flexibility and control, but this sometimes requires additional research and effort to achieve the desired outcome.
 
-For more details, see the [logs information in our best practices guide](/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-concepts#logs).
+For a closer look at the differences between OpenTelemetry and New Relic instrumentation, see our [detailed comparison](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-comparison).
 
-<table>
-  <thead>
-    <tr>
-      <th style={{ width: "200px" }}>
-        Feature
-      </th>
+## OpenTelemetry is continually evolving [#emerging-standard]
 
-      <th>
-        Supported
-      </th>
-
-      <th>
-        Description
-      </th>
-    </tr>
-  </thead>
-
-  <tbody>
-    <tr>
-      <td>
-        LogRecord body
-      </td>
-
-      <td>
-        ✅
-      </td>
+The OpenTelemetry project has a large scope which has grown over the years. While many core components have reached stability (including OTLP, the trace API / SDK, the metric API / SDK, the log bridge API / SDK, http semantic conventions, and many language implementations), there are naturally pieces which are at various other stages of maturity.
 
-      <td>
-        Supported types: string, boolean, int, double, bytes
-      </td>
-    </tr>
+New Relic aims to have first class support for OpenTelemetry, including ingesting all OTLP data into our general purpose observability platform and building user experiences on top of OpenTelemetry data to help drive insights from the data out of the box. As components emerge and develop our platform capabilities will evolve alongside. However, please be cognizant of the maturity status of any OpenTelemetry components you are integrating with. While we try to stay on top of changes, it can be challenging to build around breaking changes of experimental components.
 
-    <tr>
-      <td>
-        LogRecord attributes
-      </td>
+## OpenTelemetry reference architecture [#reference-architecture]
```

Comment:
I moved the ref architecture page to a section here. The architecture diagram itself still needs some updating, but even in its current state, I think its useful to put a picture to words.